### ISSUE TITLE
What about 1060 6GB 

### ISSUE THREAD

**Issue by lutsifer** [Optiminer/OptiminerZero#9]
Having a version for 6GB will be of huge advantage... you can't imagine how many people use P106 100 :)

**Comment by JimTriangle** [comment]
Yeah, waiting for it too ! 10 GTX 1060 waiting for version 6GB :)

**Comment by Optiminer** [comment]
You can try this test version on 6GB cards:
https://www.dropbox.com/s/urw4j7oue34hiqe/optiminer-zero-1.2-test.tar.gz?dl=1

**Comment by lutsifer** [comment]
Yes, it works! ethOS

**Comment by JimTriangle** [comment]
At the moment i'm on windows and haven't installed a linux part. I'll try it when it will be done :) Thanks
